src/structure/widget.ts

```diff
@@ -179,7 +180,15 @@ export class MoleculeViewer {
         this.guid = 'chsp-' + guid;
 
         this._root = document.createElement('div');
-        const root = getByID(id);
+
+        let root;
+        if (typeof element !== 'string') {
+            assert(element instanceof HTMLElement);
+            root = element;
+        } else {
+            root = getByID(element);
+        }
+
```

Comment:
Since this is repeated multiple times, should we make it into its own function? Something like

```typescript
function getElement(element: string | HTMLElement) {
    if (typeof element !== 'string') {
        assert(element instanceof HTMLElement);
        return element;
    } else {
        return getByID(element);
    }
}
```

Reply:
Good call.